gobblin-config-management/src/main/java/gobblin/config/configstore/ConfigStoreWithImportedByRecursively.java

```diff
@@ -0,0 +1,26 @@
+package gobblin.config.configstore;
+
+import java.net.URI;
+import java.util.Collection;
+import java.util.Map;
+
+import com.typesafe.config.Config;
+
+public interface ConfigStoreWithImportedByRecursively {
```

Comment:
Prefer to have another interface as we have two interfaces for getConfig and getResolvedConfig